ludwig/features/base_feature.py

```diff
@@ -17,6 +17,8 @@
 from abc import ABC, abstractmethod
 from typing import Dict
 
+import dask.dataframe as dd
```

Comment:
Because Dask is optional and this module is always imported, we need to guard this import similar to how we do it [here](https://github.com/ludwig-ai/ludwig/blob/master/ludwig/utils/data_utils.py#L37):

```
try:
    import dask.dataframe as dd
except ImportError:
    pass
```

For the purposes of type checking, this should be fine.

Reply:
Thanks for pointing this out Travis! Just fixed this. I went with the first approach you listed.